integration-tests/utils/integration-utils.js

```diff
@@ -111,6 +111,10 @@ const findText = (page, textContains, options) => {
   return page.waitForXPath(`//*[contains(normalize-space(.),"${textContains}")]`, options)
 }
 
+const findTooltipText = (page, textContains, options = { visible: true }) => {
+  return page.waitForXPath(`//*[@id="modal-root"]//*[@role="tooltip" and contains(normalize-space(.),"${textContains}")]`, options)
+}
```

Comment:
I see. 
It also doesn't seem to be used anywhere?

Reply:
I edited test so it no longer does hover then check tooltip text because hover seems to be unreliable (good thing that I caught this before merge). I left this func alone because I think it can be useful in future to others. Do you want it removed?

Reply:
I'd suggest removing as a lot of what I had thought would be needed didn't end up being needed :)